lib/net/peerpool.js

```diff
@@ -185,6 +188,28 @@ class PeerPool extends EventEmitter {
       }
     }
   }
+
+  /**
+   * Peer pool status check on a repeated interval
+   */
+  async _statusCheck () {
+    if (this.size === 0) {
+      this.noPeerPeriods += 1
+      if (this.noPeerPeriods >= 3) {
+        this.servers.map(async server => {
```

Comment:
```ts
cost promises = this.servers.map(async server => {])
await Promise.all(promises)
```

Reply:
good catch, thanks! i added this in https://github.com/ethereumjs/ethereumjs-client/pull/133/commits/f748a18fc0103990276b3227bd5ff4ee6ba31e3e

Reply:
Thanks!